### ISSUE TITLE
Low priority, but www.bic-unblocked.org doesn't resolve

### ISSUE THREAD

**Issue by lewlefton** [UnBlockEd-Project/unblocked-site#13]
May want to have this be a cname for bic-unblocked.org 

This is not something that can be addressed in this project, but just putting it out there in case @haydenfree can make the change on the hosting platform's DNS.

**Comment by haydenfree** [comment]
Perfect place to put the issue, and I'll fix this today.

**Comment by haydenfree** [comment]
Should be fixed, can you confirm?

www.bic-unblocked.org

**Comment by lewlefton** [comment]
Looks good.  Still needs SSL cert

**Comment by haydenfree** [comment]
https://bic-unblocked.org and https://www.bic-unblocked.org should both be working with SSL
